platforms/quorum/configuration/samples/network-minikube.yaml

```diff
@@ -0,0 +1,215 @@
+  # to join an existing network.
+  type: quorum
+  version: 2.1.1 #this is the version of Quorum docker image that will be deployed
+
+  #Environment section for Kubernetes setup
+  env:
+    type: "local" # tag for the environment. Important to run multiple flux on single cluster
+    proxy: ambassador # value has to be 'ambassador' as 'haproxy' has not been implemented for Quorum
+    ## Any additional Ambassador ports can be given below, must be comma-separated without spaces, this is valid only if proxy='ambassador'
+    #  These ports are enabled per cluster, so if you have multiple clusters you do not need so many ports
+    #  This sample uses a single cluster, so we have to open 4 ports for each Node. These ports are again specified for each organization below
+    ambassadorPorts: 15010,15011,15012,15013,15020,15021,15022,15023,15030,15031,15032,15033,15040,15041,15042,15043
+    retry_count: 20 # Retry count for the checks on Kubernetes cluster
+    external_dns: enabled # Should be enabled if using external-dns for automatic route configuration
+
+  # Docker registry details where images are stored. This will be used to create k8s secrets
+  # Please ensure all required images are built and stored in this registry.
+  # Do not check-in docker_password.
+  docker:
+    url: "index.docker.io/hyperledgerlabs"
+    username: "<YOUR DOCKER USERNAME>"
+    password: "<YOUR DOCKER PASSWORD>"
+
+  # Following are the configurations for the common Quorum network
+  config:
+    consensus: "raft" # Options are "raft" and "ibft"
+    ## Certificate subject for the root CA of the network.
+    #  This is for development usage only where we create self-signed certificates and the truststores are generated automatically.
+    #  Production systems should generate proper certificates and configure truststores accordingly.
+    subject: "CN=DLT Root CA,OU=DLT,O=DLT,L=London,C=GB"
+    transaction_manager: "tessera" # Options are "tessera" and "constellation"
+    tm_version: "0.9.2" # This is the version of "tessera" and "constellation" docker image that will be deployed
+    tm_tls: "strict" # Options are "strict" and "off"
+    tm_trust: "tofu" # Options are: "whitelist", "ca-or-tofu", "ca", "tofu"
+    ## Transaction Manager nodes public addresses should be provided.
+    #  For "tessera", all participating nodes should be provided
+    #  For "constellation", only one is bootnode should be provided
+    #
+    # For constellation, use following. This will be the bootnode for all nodes
+    #  - "http://carrier.test.quorum.blockchaincloudpoc.com:15013"
+    # The above domain name is formed by the http://(peer.name).(org.external_url_suffix):(ambassador constellation port)
+    # In the example (for tessera ) below, the domain name is formed by the https://(peer.name).(org.external_url_suffix):(ambassador default port)
+    tm_nodes:
+      - "https://carrier.test.quorum.blockchaincloudpoc.com:8443"
```

Comment:
These addresses will not work for tessera. The addresses need to be local kubernetes address.